pkg/app/piped/executor/kubernetes/rollback.go

```diff
@@ -171,3 +179,45 @@ func (e *rollbackExecutor) ensureRollback(ctx context.Context) model.StageStatus
 	}
 	return model.StageStatus_STAGE_SUCCESS
 }
+
+func (e *rollbackExecutor) ensureScriptRunRollback(ctx context.Context) model.StageStatus {
+	e.LogPersister.Infof("Runnnig commands for rollback...")
+
+	onRollback, ok := e.Stage.Metadata["onRollback"]
+	if !ok {
+		e.LogPersister.Error("onRollback metadata is missing")
+		return model.StageStatus_STAGE_FAILURE
+	}
+
+	if onRollback == "" {
+		e.LogPersister.Info("No commands to run")
+		return model.StageStatus_STAGE_SUCCESS
+	}
```

Comment:
@ffjlabo What I mean here is that we may make this:
1. Add rollbackable of type bool to specify whether the current script run can be rollbacked or not, default set to false 🤔 
2. In case of rollbackable stage, if `onRollback` is not set, re-run script instead or return error rollback script is missing
wdyt?

Reply:
On the second thought, I got your idea.
- If users did not define `onRollback` script, then there is nothing to run
- If users defined the `onRollback` script, then run that script
Let's keep this idea, thanks 👍